release-notes.md

```diff
@@ -12,6 +12,14 @@ The following change was made in this release:
 
 * Added the ability to reset the `pkey`.
 
+**iOS Analytics 2.2.1**
+
+* Analytics response content events now contain two new fields: `hitHost` and `hitUrl`. These fields contain the host and url of the of the hit responsible for dispatching the response event.
```

Comment:
Can you please put the field names as a bulleted list? Like this:

Analytics response content events now contain two new fields: 
* `hitHost`
`hitUrl`. 
These fields contain the host and URL of the hit responsible for dispatching the response event.

(change also applies to the next section)

Reply:
done :)